source/includes/_personnel.md

```diff
@@ -180,6 +193,7 @@ email | String | No | Email address of the personnel
 external_id | String | No | External ID of the personnel
 line_manager_id | String | No | ID of the line manager of the personnel
 role_ids | Association IDs | No | IDs of roles that exist in the training register, that the personnel holds. For more information see [Associations](#associations)
+primary_role_id | String | No | ID of the primary role of the personnel
```

Comment:
perhaps this should be moved above `more information see [Associations](#associations)`